Components/GridPostsTeaser/_style.scss

```diff
@@ -0,0 +1,148 @@
+[is='flynt-grid-posts-teaser'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 4.375rem;
+    text-align: center;
+  }
+
+  .postContent {
+    margin-top: $container-padding-mobile + 10;
+    text-align: center;
+
+    @media (min-width: $breakpoint-tablet) {
+      margin-top: $container-padding-tablet + 10;
+    }
+
+    @media (min-width: $breakpoint-desktop) {
+      margin-top: $container-padding-desktop + 10;
+    }
+  }
+
+  .grid {
+    @include reset-list;
+    @include grid(1, center);
+
+    @for $i from 2 through 4 {
+      &--length#{$i} {
+
+        @media (min-width: $breakpoint-tablet) {
+          @include grid(2, center);
+        }
+
+        @media (min-width: $breakpoint-desktop) {
+          @include grid($i, center);
+        }
+      }
+    }
+
+    &-item {
+      display: flex;
+      max-width: $content-max-width;
+    }
+  }
+
+  .content {
+    box-shadow: 0 1px 26px 0 rgba(0, 0, 0, 0.06);
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    text-decoration: none;
+    width: 100%; // IE11 bug fix
+
+    @media (min-width: $breakpoint-mobile-horizontal) {
+      transition: box-shadow 0.2s ease-in;
+    }
+
+    &:hover {
+      @media (min-width: $breakpoint-mobile-horizontal) {
+        box-shadow: none;
+      }
+    }
+
+    &-image {
+      flex-shrink: 0; // IE11 bug fix
+    }
+
+    &-inner {
+      background-color: $color-background;
+      flex-grow: 1;
+      padding: 1.25rem;
+    }
+
+    &-title {
+      color: $color-headline;
+      padding-top: 0;
+    }
+
+    &-date {
+      align-items: center;
+      display: flex;
+      margin: 0;
+
+      dt,
+      dd {
+        color: #7b838d;
+        margin: 0;
+      }
+
+      dt {
+        display: inline-block;
+        line-height: 1rem;
+        margin-right: 0.375rem;
+
+        .feather {
+          color: inherit;
+          height: 1rem;
+          top: 0;
+          width: 1rem;
+        }
+      }
+    }
+
+    &-header {
+      @include reset-list;
+
+      align-items: center;
+      display: flex;
+      font-size: 0.8125rem;
+      justify-content: space-between;
+      margin-bottom: 0.625rem;
+    }
+
+    &-pill {
```

Comment:
Could / should this be a global base style class?

Reply:
This should definitely be a global class.
In fact, in my opinion, all the elements that are commonly used a cross the web site should be defined in global scope.
The logic is, if we can use a `button` class in global scope, what is the reason not to use other elements?
For example, card-looking element is being re-defined in every component even though all the instances of it share quite a bit of common properties, like the `padding` and `background-color` (please see screenshot).
---
I would not use the base style file for this, but rather block-specific files: `buttons.scss, pills.scss, etc.`

![glob](https://user-images.githubusercontent.com/49150551/61520913-7bd07080-aa0f-11e9-8cbd-6ece9e4412b7.png)